pkg/mcs/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package mcs provides utilities and extensions for Kubernetes Multi-cluster
+// Services API as defined by KEP 1645. 
+// (https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
+package mcs
```

Comment:
Not sure package name `mcs` is descriptive enough.
Agent part is also part of our mcs implementation.
The whole lighthouse is, actually.
Isn't in better to call it `hub`?

Reply:
Oh I see, its a package for shared code between spoke and hub.